js/utilities.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const rand = (min, max) => {
+  return Math.floor(Math.random() * (max - min)) + min;
+}
+
+const getRandomPoint = (min, max) => {
```

Comment:
The grid might not be a square, you should pick a point between differents min / max for the x and the y coordinates.